### PR TITLE
AOM-122: Check for Updates returns No updates found

### PR DESCRIPTION
**Pull request by aenkya**
## JIRA TICKET NAME:
[AOM-122: Check for Updates returns No updates found](https://issues.openmrs.org/browse/AOM-122)

### SUMMARY:
Clicking the check for updates button returns no updates found despite the presence of updates.
It should list modules with updates available

### CODE REVIEW

File: app/js/components/manageApps/ManageApps.jsx [hunk 1]
```diff
@@ -167,7 +168,7 @@ export default class ManageApps extends React.Component {
         });
         installedAddons = installedOwas.concat(installedModules);
         return Promise
-          .all(response.data.results.map(data => axios.get(`${ApiHelper.getAddonUrl()}?modulePackage=${data.packageName}`)
+          .all(response.data.results.map(data => axios.get(`${ApiHelper.getAddonUrl()}/${data.packageName}`)
```
Comment (segunolalive) [inline app/js/components/manageApps/ManageApps.jsx:171]: This change reverses the effect of `AOM-113`. Is there a reason for that?

**Review by segunolalive — COMMENTED** [review]
looks good.

File: app/js/components/manageApps/ManageApps.jsx (hunk 1 above)
Comment (aenkya) [reply app/js/components/manageApps/ManageApps.jsx:171]: @segunolalive the previous URL request causes the request for updates to return no updates found. The issue it tries to resolve is less critical and can be adjusted with a UX/UI implementation adjustment

Comment (Charpell) [reply app/js/components/manageApps/ManageApps.jsx:171]: The URL i used for retrieving the developer's name in `AOM-103` has somewhat broken the `check for update link` and created other bugs in the app. So like @Elbertbiggs360 suggested, this PR takes priority for now and we will try to fix resolve `AOM-103` using a UX/UI implementation

Comment (segunolalive) [reply app/js/components/manageApps/ManageApps.jsx:171]: That's okay.

Comment (dkayiwa) [reply app/js/components/manageApps/ManageApps.jsx:171]: So can you reopen that ticket and give the author these details? As in tell them why you have reversed their change?

Comment (aenkya) [reply app/js/components/manageApps/ManageApps.jsx:171]: Alright, thanks

**Pull request merged**